XENONnT/GOFevaluation
# Example Notebook

This PR adds an example notebook that can be used as a guide for using the package for the first time.

Additionally a few more small changes:
* Improve the `__str__` method of the `GOFTest`class so that `print(gof_test)` provides a more readable result like:

  >  GOFevaluation.gof_test
  >  GOF measures: ADTestTwoSampleGOF, KSTestTwoSampleGOF, PointToPointGOF
  > 
  >   **ADTestTwoSampleGOF**
  >   gof = 1.6301454042304904
  >   p-value = 0.08499999999999996
  > 
  >   **KSTestTwoSampleGOF**
  >   gof = 0.13999999999999996
  >   p-value = 0.09799999999999998
  > 
  >   **PointToPointGOF**
  >   gof = -0.7324060759792504
  >   p-value = 0.128
* Add mybinder to the repo. This way one can give the package a try without installing it
* Include coveralls to monitor code coverage (and improve it in the future)
* Make linting a bit pickier to make sure code remains readable in the future (and remove my formatting sins from the past..)
* Update the readme
